src/models/user.js

```diff
@@ -1,34 +1,27 @@
-/* eslint-disable linebreak-style */
 const mongoose = require('mongoose');
 
-UserSchema = new mongoose.Schema(
+const { Schema } = mongoose;
 
-{
-  Name:{
+const UserSchema = new Schema({
```

Comment:
can we have another column called verified_at and active.
The verified_at will be for users that signup with email and needs verification while active will be used to block and unblock users